Packs/Slack/doc_files/SlackV3_app_manifest.yml

```diff
@@ -31,8 +31,6 @@ oauth_config:
       - groups:read
       - groups:write
       - im:history
-      - im:read
-      - im:write
```

Comment:
Did you verify that all the commands work? Specifically send-notification to a user?

Reply:
send-notification still works yes, do you know why these permissions were there in the first place? According to slack these are to give the bots ability to read and write in dms they are added to

Reply:
I believe when I set it up a while back, it required those permissions in order for the bot to send a DM to someone. If it works without, then let's remove.